### ISSUE TITLE
Change the name of `Lawyer office`

### ISSUE THREAD

**Issue by Dimitar5555** [openstreetmap/id-tagging-schema#300]
Currently `office=lawyer + lawyer=notary` and `office=lawyer` have the same name (`Notary office`) which can cause confusion if the translator doesn't see the tags. Is it possible to change the preset name of `office=lawyer` to something else?

**Comment by tyrasd** [comment]
You mean that `office=notary` shares the same name and should be renamed, right?

The [wiki](https://wiki.openstreetmap.org/wiki/Tag:office%3Dnotary) is not very helpful on this matter, unfortunately:

> Note that this tag may have different meaning (currently it is not defined) […]

Note that currently the preset for `office=lawyer + lawyer=notary` is a "hidden" one (not searchable), so it wouldn't cause confusion by being a duplicate in the search results.

What would you propose to call these two presets?

**Comment by Dimitar5555** [comment]
I meant `office=notary`.
`office=notary` can be called `Notary office` and `office=lawyer + lawyer=notary` can be called `Civil law notary` (as they are called in the respective Wikipedia articles). I'm not sure if it would make sense when it's translated in other languages though.

**Comment by tyrasd** [comment]
I found some older tickets regarding this tag:

* https://github.com/openstreetmap/iD/issues/4634
* https://github.com/openstreetmap/iD/pull/4491#commitcomment-25316574

and here's a recent [usage statistics graph](https://taghistory.raifer.tech/#***/office/notary&***/lawyer/notary):

[![comparison of usage of the discussed tags over time](https://user-images.githubusercontent.com/1927298/149755704-6afd27d3-3cc2-414b-8768-b7a304e7de6c.png)](https://taghistory.raifer.tech/#***/office/notary&***/lawyer/notary).

**Comment by tyrasd** [comment]
I'm not really familiar with the _common law_ system, so the concept of different kinds of notaries was new for me. But after reading this topic up a bit on wikipedia, what I understand is that in some countries (the using the [common law](https://en.wikipedia.org/wiki/Common_law) system, i.e., UK, Ireland, US, Canada, Australia, New Zealand and perhaps a few others) there exist two different kinds of notaries: _notaries public_ and _civil law notaries_. In other countries (e.g., continental Europe) this distinction does not seem to exist (anymore).

Having this in mind, I think the decision to hide the preset for `lawyer=notary` seems like it was a mistake.

Now, the question is: wouldn't it be best if we limit the `office=notary` preset to countries with _common law_ systems and call it _Notary Public_?

**Comment by 1ec5** [comment]
> Now, the question is: wouldn't it be best if we limit the office=notary preset to countries with common law systems and call it Notary Public?

Yes, I think a regionally limited preset for notaries public makes plenty of sense, as long as we can reliably identify the jurisdictions where they operate. It’s possible that some of the earlier confusion was because civil law notaries are unheard of in most of the United States. But adding to the confusion, civil law notaries apparently do operate in parts of the U.S., including [Louisiana](https://lna.org/Main/What_your_Louisiana_Notary_Can_Do_for_You.aspx) (which uses civil law) and [Florida](https://dos.myflorida.com/sunbiz/other-services/notaries/civil-law-notary/) (which uses common law). So we would need to keep the civil law notary preset enabled for these hybrid jurisdictions as well.

The capability to define regional presets changes the calculus for a lot of old discussions like the ones you dug up. For example, Bail Bond Agent could also be limited to certain legal systems, but that probably isn’t causing any problems because “bail bond agent” doesn’t mean a different thing in other legal systems.

**Comment by matkoniecz** [comment]
> The [wiki](https://wiki.openstreetmap.org/wiki/Tag:office%3Dnotary) is not very helpful on this matter, unfortunately:

I tried to understand what this issue is about and wiki for me failed to answer clearly: 

where this different notaries are different?

what is the difference ("notary public is different from a civil law notary" is not really clear)?

Why `office=notary` is only for common law countries?

If anyone cares about this topic it would be useful to improve documentation.
